[2022-09-04T16:28:02.045+0000] {taskinstance.py:1171} INFO - Dependencies all met for <TaskInstance: first_airflow_dag.get_datetime manual__2022-09-04T16:28:01.284800+00:00 [queued]>
[2022-09-04T16:28:02.054+0000] {taskinstance.py:1171} INFO - Dependencies all met for <TaskInstance: first_airflow_dag.get_datetime manual__2022-09-04T16:28:01.284800+00:00 [queued]>
[2022-09-04T16:28:02.056+0000] {taskinstance.py:1368} INFO - 
--------------------------------------------------------------------------------
[2022-09-04T16:28:02.057+0000] {taskinstance.py:1369} INFO - Starting attempt 1 of 1
[2022-09-04T16:28:02.059+0000] {taskinstance.py:1370} INFO - 
--------------------------------------------------------------------------------
[2022-09-04T16:28:02.071+0000] {taskinstance.py:1389} INFO - Executing <Task(BashOperator): get_datetime> on 2022-09-04 16:28:01.284800+00:00
[2022-09-04T16:28:02.076+0000] {standard_task_runner.py:52} INFO - Started process 3501 to run task
[2022-09-04T16:28:02.080+0000] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'first_***_dag', 'get_datetime', 'manual__2022-09-04T16:28:01.284800+00:00', '--job-id', '66', '--raw', '--subdir', 'DAGS_FOLDER/first_dag.py', '--cfg-path', '/tmp/tmpzwqaec6a', '--error-file', '/tmp/tmpsn96flzp']
[2022-09-04T16:28:02.083+0000] {standard_task_runner.py:80} INFO - Job 66: Subtask get_datetime
[2022-09-04T16:28:02.157+0000] {task_command.py:371} INFO - Running <TaskInstance: first_airflow_dag.get_datetime manual__2022-09-04T16:28:01.284800+00:00 [running]> on host c56acb3a0493
[2022-09-04T16:28:02.202+0000] {taskinstance.py:1583} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=first_***_dag
AIRFLOW_CTX_TASK_ID=get_datetime
AIRFLOW_CTX_EXECUTION_DATE=2022-09-04T16:28:01.284800+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-09-04T16:28:01.284800+00:00
[2022-09-04T16:28:02.204+0000] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-09-04T16:28:02.206+0000] {subprocess.py:74} INFO - Running command: ['/bin/bash', '-c', 'date']
[2022-09-04T16:28:02.217+0000] {subprocess.py:85} INFO - Output:
[2022-09-04T16:28:02.219+0000] {subprocess.py:92} INFO - Sun Sep  4 16:28:02 UTC 2022
[2022-09-04T16:28:02.220+0000] {subprocess.py:96} INFO - Command exited with return code 0
[2022-09-04T16:28:02.242+0000] {taskinstance.py:1412} INFO - Marking task as SUCCESS. dag_id=first_***_dag, task_id=get_datetime, execution_date=20220904T162801, start_date=20220904T162802, end_date=20220904T162802
[2022-09-04T16:28:02.295+0000] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-09-04T16:28:02.324+0000] {local_task_job.py:279} INFO - 1 downstream tasks scheduled from follow-on schedule check
